### DISCUSSION TITLE
Import on iOS

### DISCUSSION THREAD

**Discussion by BlockBuilderJoe** [guillaumechereau/goxel#369] [Q&A]
I can't see a way to import on iOS like you can on Linux is it possible?

**Comment by YarlBoro** [comment]
It definitely is possible. Put voxel file that you want to import inside Goxels iCloud folder. After that restart Goxel - so it refreshes its file browser - and open your model. If you want to import something else than *.gox file, like *.vox or *.qb procedure is the same. You just need to highlight the file and press button with pen icon.

Don't forget to save imported file before closing Goxel, otherwise you will loose changes.

**Reply by BlockBuilderJoe** [reply]
Thanks for your help, it's working  perfectly. 

In case this helps anyone, I didn't have an iCloud folder for Goxel by default, but I got one by enabling it in settings and then exporting a model which created the folder.

**Comment by guillaumechereau** [comment]
I think I am also going to add an 'import' button in the mobile app, to open any file from the phone.
